metadata/cmd/pilot/tls-config/Makefile

```diff
@@ -37,7 +37,7 @@ pilot-client.key:
 
 pilot-client.crt: pilot-client.key openpitrix-ca.key openpitrix-ca.crt
 	openssl req -new \
-		-subj "/C=GB/L=China/O=pilot-client/CN=pilot-client.openpitrix.io" \
```

Comment:
the "pilot.openpitrix.io" is pilot server name, client common name is different.